server/src/test/java/io/crate/integrationtests/JoinIntegrationTest.java

```diff
@@ -1108,4 +1110,16 @@ public void test_many_table_join_with_filter_pushdown() throws Exception {
         execute(stmt);
         assertThat(response.rowCount(), is(0L));
     }
+
+    @Test
+    public void test_ambiguous_columns() {
```

Comment:
Could we maybe come up with a unit test for this? I think adding integration tests for error code mapping will slow down the test suite to much for what is being tested.

Reply:
There are a couple of unit tests that assert for `AmbiguousColumnException`

Maybe we could extend the assertion there to also verify the mapped errors.
We could have some kind of

`assertThrows(... , errors(orginalExcpetion, mappedPsqlError))`

Reply:
Or alternatively we could write a dedicated unit test for the mapping logic. That way we wouldn't clutter all the other unit tests with additional mapping information and we'd still have the integration level tests where we confirm both http and postgresql error codes.

Reply:
Ok, thanks for feedback. I will see what I can do.